languages/r36/template/src/Algorithm.r

```diff
@@ -0,0 +1,36 @@
+library(algorithmia)
+# API calls will begin at the algorithm() method, with the request body passed as 'input'
+# For more details, see algorithmia.com/developers/algorithm-development/languages
+
+algorithm <- function(input) {
+  paste("hello", input)
+}
+
+# Here is an example of an advanced form of an algorithm function, using a load function (like the example below)
```

Comment:
there are a ton of comments in here. To me it seems a bit more complicated this way

Reply:
was a request by @StephanieKim, I'm indifferent haha